CONTRIBUTING.md

```diff
@@ -1,5 +1,29 @@
-# Bug Reports & Feature Requests
+# List of rules and norms
+
+## Bug Reports & Feature Requests
 Bug reports and feature requests are greatly appreciated and can be done in form of an [issue](https://github.com/JakeRoggenbuck/auto-clock-speed/issues)
 
-# Develop
+## Develop
 If you would like to help complete a task, comment in the issue and we may assign it to you!
+
+## Pull Requests
+- Pull requests must be tested but with `cargo test` and ran on local machine (laptop with battery)
+- Pull requests that change core functionality must have a review
```

Comment:
Maybe change that to all PRs must have at least 1 review unless it's by jake cuz jakes cool